### PR TITLE
Minor optimizations to cmd_print.c

### PR DESCRIPTION
**Pull request by eagleoflqj**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -6946,8 +6941,7 @@ R_API void r_print_offset_sg(RPrint *p, ut64 off, int invert, int offseg, int se
 			} else {
 				if (offdec) {
 					snprintf (space, sizeof (space), "%"PFMT64u, off);
-					white = r_str_pad (' ', 10 - strlen (space));
-					r_cons_printf ("%s%s", white, space);
+					r_cons_printf ("%10s", space);
```
Comment (trufae) [inline libr/core/cmd_print.c:6944]: Align left vs align right?

**Review by trufae — CHANGES_REQUESTED** [review]

File: libr/core/cmd_print.c (hunk 1 above)
Comment (eagleoflqj) [reply libr/core/cmd_print.c:6944]: As https://en.cppreference.com/w/c/io/fprintf says, without `-` after `%`, align right is applied.
```
-: the result of the conversion is left-justified within the field (by default it is right-justified)
(optional) integer value or * that specifies minimum field width. The result is padded with space characters (by default), if required, on the left when right-justified, or on the right if left-justified. In the case when * is used, the width is specified by an additional argument of type int. If the value of the argument is negative, it results with the - flag specified and positive field width. (Note: This is the minimum width: The value is never truncated.)
```

Comment (XVilka) [reply libr/core/cmd_print.c:6944]: We should check this behavior on Windows too, just in case, before merging. cc @GustavoLCR @kazarmy @pelijah @a1ext

Comment (eagleoflqj) [reply libr/core/cmd_print.c:6944]: FYI https://rextester.com/l/c_online_compiler_visual

Comment (XVilka) [reply libr/core/cmd_print.c:6944]: Yes, it works as intended.

**Review by XVilka — APPROVED** [review]

**Review by wargio — APPROVED** [review]
LGTM.

**Review by trufae — APPROVED** [review]

**Comment by XVilka** [comment]
@ret2libc please take one more look, if you are good - lets merge.

**Review by ret2libc — APPROVED** [review]

**Pull request merged**